### PR TITLE
Fix file paths in Notebook.ipynb

### LINKED ISSUE #175: Notebook.ipynb fails to execute
In notebook `examples/Notebook.ipynb`, this cell fails to execute:
```py
from jupytercad import CadDocument

doc = CadDocument("examples/example3.FCStd")

doc
```
image
```
[E 2023-06-02 15:40:05.639 YDocExtension] Error watching file: examples/examples/example3.FCStd
    HTTPError()
    Traceback (most recent call last):
      File "/home/david/micromamba/envs/jupytercad/lib/python3.9/site-packages/jupyter_collaboration/loaders.py", line 170, in _watch_file
        await self.notify()
      File "/home/david/micromamba/envs/jupytercad/lib/python3.9/site-packages/jupyter_collaboration/loaders.py", line 183, in notify
        model = await ensure_async(self._contents_manager.get(path, content=False))
      File "/home/david/micromamba/envs/jupytercad/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 182, in ensure_async
        result = await obj
      File "/home/david/micromamba/envs/jupytercad/lib/python3.9/site-packages/jupyter_server/services/contents/filemanager.py", line 877, in get
        raise web.HTTPError(404, "No such file or directory: %s" % path)
    tornado.web.HTTPError: HTTP 404: Not Found (No such file or directory: examples/examples/example3.FCStd)
```

### PR DESCRIPTION
**Pull request by davidbrochart**
Closes #175

### CODE REVIEW

File: examples/Notebook.ipynb [hunk 1]
```diff
@@ -133,7 +133,7 @@
    "source": [
     "from jupytercad import CadDocument\n",
     "\n",
-    "doc = CadDocument(\"examples/example3.FCStd\")\n",
```
Comment (trungleduc) [inline examples/Notebook.ipynb:136]: This path depends on where you start `jupyterlab`

Comment (martinRenou) [reply examples/Notebook.ipynb:136]: It used to but it should have been fixed in JupyterLab 4.0.1 https://github.com/jupyterlab/jupyterlab/commit/0d7a24c86dcd328d3d2a59ad2e736c5ea71a81b3

Comment (trungleduc) [reply examples/Notebook.ipynb:136]: ah ok, nice!!!

**Comment by martinRenou** [comment]
So Trung's comment makes me think we should update the jupyterlab dependency to >=4.0.1 to make sure to include the fix

**Review by martinRenou — APPROVED** [review]
Thanks!

**Comment by trungleduc** [comment]
> Sounds good :P

Actually, we are, I'll bump it in another PR

**Pull request merged**